src/App.tsx

```diff
@@ -174,7 +174,8 @@ export default function App() {
 		<div id={"output-area"}>
 			<SelectionInfoView markerData={markerFileData} selectedMarkers={selectedMarkers} frame={frame} />
 		</div>
-		{/* ---------------------------------------------- Grid Row 3 ---------------------------------------------- */}
+		<img id={"sdp-logo"} src={"https://drive.google.com/uc?export=view&id=10VJZEoeTEN7JKled94_VoUMK4c_PEfZc"} alt={"senior design project logo"} />
```

Comment:
I don't think we should be loading this from Google

Reply:
Updated. There were a variety of approaches for importing local images, but I'm not sure what is idiomatic for Typescript. The solution I settled on seems to work fine, hopefully it's okay.